src/main/java/Events.java

```diff
@@ -0,0 +1,75 @@
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+public class Events extends Task {
+    private String timeDate;
+    private String interpretedDate;
+    private String time;
+
+    /**
+     * @param task task of the Events object
+     * @param timeDate time and date of the Event object in the following format: 2/12/2019 2-4pm
+     *
+     */
+
+    public Events(String task, String timeDate) {
+        super(task);
+        this.timeDate = timeDate;
+        interpretDate();
+    }
+
+    /**
+     * returns the message comprising [E], name of the task of the Events object and its date and start to end time,
+     * together with the a symbol representing if an Events object is completed or not.
+     * [E] represents that our object is a Events object.
+     *
+     */
+    public String toString() {
+        String box;
+        String msg = "[E]";
+        if (!this.isDone()) {
+            box = "[✗]";
+        } else {
+            box = "[✓]";
+        }
+        return msg + box + " " + this.getTask() + " (at: " + interpretedDate + " from " + time + ")";
+    }
+
+    /**
+     * interprets date of Events object from timeDate attribute.
+     *eg 2/12/2019 becomes 2 December 2019
+     *2-4pm stays the same
+     */
+
+    public void interpretDate() {
```

Comment:
Consider having a DateUtil class with a static DateUtil.parse(String) method that returns a Date.